lib/cmr/cmr_client.rb

```diff
@@ -115,6 +115,17 @@ def get_granule_count(collection_id, token)
       get(url, options, token_header(token))
     end
 
+    # This method is to retrieve the collection for xml download
+    def get_collection_concept_for_download(concept_id, collection_format, token, revision_id = nil)
```

Comment:
What is different between this and the `get_concept` method? Seems like we can just allow the ability to pass query params (to support pretty=true)

Reply:
There is not much difference aside from the extension. At the time I thought it might be helpful to keep it separate, but it does seem unnecessary.